### ISSUE TITLE
Failure when runing pytest

### ISSUE THREAD

**Issue by nguyen-tuonghuy** [fplll/g6k#50]
Hi,

I installed G6K using conda, following https://martinralbrecht.wordpress.com/2020/08/23/conda-jupyter-and-emacs/.

The installation goes as expected, but when running tests :

> python -m pytest

I happened to have a failed test at "siever.pyc" :

> _________________________________________________________________________ [doctest] siever.pyx __________________________________________________________________________
1569 
1570             >>> g6k = Siever(A)
1571             >>> g6k.initialize_local(0, 10, 30)
1572             >>> g6k.best_lifts()
1573             []
1574 
1575             >>> g6k()
1576             >>> bl = g6k.best_lifts()
1577             >>> id, nrm, w = bl[0]
1578             >>> id, round(nrm)
Expected:
    (0, 194629.0)
Got:
    (0, 194629)
/home/g6k/g6k/siever.pyx:1578: DocTestFailure

Apart from this failed test, the other computations from the readme file seem to run as expected.

**Comment by lducas** [comment]
I am experiencing the same issue even without conda. @malb I'm happy to just change the doctest, but I am worried about the source of the change of behaviour. Python 2 vs Python 3 maybe ?

**Comment by lducas** [comment]
Indeed, this is Python 2 VS Python 3.
```
> python2
Python 2.7.18rc1 (default, Apr  7 2020, 12:05:55) 
>>> round(0.1)
0.0
> python3
Python 3.8.5 (default, Jul 28 2020, 12:59:40) 
>>> round(0.1)
0
```
Fixed by #54
